选修课程/计算机视觉/HITCV_EXP/exp3/exp-03-code/KAIR/models/basicblock.py

```
@@ -0,0 +1,591 @@
+        elif t == 'U':
+            L.append(nn.Upsample(scale_factor=2, mode='nearest'))
+        elif t == 'u':
+            L.append(nn.Upsample(scale_factor=3, mode='nearest'))
+        elif t == 'v':
+            L.append(nn.Upsample(scale_factor=4, mode='nearest'))
+        elif t == 'M':
+            L.append(nn.MaxPool2d(kernel_size=kernel_size, stride=stride, padding=0))
+        elif t == 'A':
+            L.append(nn.AvgPool2d(kernel_size=kernel_size, stride=stride, padding=0))
+        else:
+            raise NotImplementedError('Undefined type: '.format(t))
```

Ruff (0.15.21)

[error] `.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)